src/serval.py

```diff
@@ -1088,10 +1088,10 @@ def serval():
 
    if skyfile:
       if skyfile=='auto' and getattr(inst, 'skyfile', None):
-         skyfile = servallib + getattr(inst, 'skyfile', None)
-         sky = np.genfromtxt(skyfile)
-         skymsk = interp(lam2wave(sky[:,0]), sky[:,1])
-
+         skyfile = getattr(inst, 'skyfile', None)
+   if skyfile:
+      sky = np.genfromtxt(servallib + skyfile)
```

Comment:
It seems now  `skyfile` must be in servallib.

Reply:
Skyfile needs to be in servallib, this ensures it is analogues to the keyword in inst, where the skyfile also needs to be in servallib. Alternatively, one could  first try to find the skyfile in servallib and otherwise interpret it as dir?